### PR TITLE
Add missing groupId inside list glossaries/projects

### PR DESCRIPTION
**Pull request by dhoko**
Hey :wave: 

There was a missing `groupId` for 2 GET calls, It didn't match the API definition for:

- [projects](https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.getMany)
- [glossaries](https://developer.crowdin.com/enterprise/api/v2/#operation/api.glossaries.getMany)

### CODE REVIEW

**Review by andrii-bodnar — APPROVED** [review]
@dhoko thank you!

**Comment by andrii-bodnar** [comment]
@dhoko Unit tests have failed. Could you please take a look? 

https://github.com/crowdin/crowdin-api-client-python/actions/runs/4799498715

**Comment by dhoko** [comment]
@andrii-bodnar :heavy_check_mark: btw there is an issue with the coverage. When I install the deps it fails at `typed_ast` :/  (Fedora 37)

``` 
│Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -m64 -mtune
│=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -fPIC -fwrapv -O2 -fex
│ceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERT
│IONS -fstack-protector-strong -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protec
│tion -D_GNU_SOURCE -fPIC -fwrapv -O2 -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,
│-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -m64 -mtune=generic -fasynchronous-unwind-ta
│bles -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -fPIC -fwrapv -fPIC -Iast27/Include -I/home/dhoko/.loca
│l/share/virtualenvs/crowdin-api-client-python-1Vpfnig1/include -I/usr/include/python3.11 -c ast27/Custom/typed_ast.
│c -o build/temp.linux-x86_64-cpython-311/ast27/Custom/typed_ast.o
│      ast27/Custom/typed_ast.c:1:10: fatal error: Python.h: No such file or directory
│          1 | #include "Python.h"
│            |          ^~~~~~~~~~
│      compilation terminated.
│      error: command '/usr/bin/gcc' failed with exit code 1
│      [end of output]
│
│  note: This error originates from a subprocess, and is likely not a problem with pip.
│error: legacy-install-failure
│
│× Encountered error while trying to install package.
│╰─> typed_ast
``` 

Then we can't run pytest unless we comment the coverage cf:
```diff
diff --git a/setup.cfg b/setup.cfg
index fdbb3b3..7221536 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -60,10 +60,6 @@ addopts =
     --strict-markers
     --tb=short
     --doctest-modules
-    --cov=crowdin_api
-    --cov-report=term-missing:skip-covered
-    --cov-report=xml
-    --cov-fail-under=95
````

**Comment by andrii-bodnar** [comment]
@dhoko thank you!

Do you have any ideas regarding this coverage error? Everything seems to work well in the CI/CD pipelines

**Comment by dhoko** [comment]
Idk ¯\_(ツ)_/¯

I saw messages asking to install python3-dev... So I went with no coverage on my computer :grin: It requires no changes and we can still test everything^^

**Comment by dhoko** [comment]
https://github.com/python/typed_ast/issues/179 

tl;dr: EOL + I'm using python 3.11 -> no support
